app/views/groups/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="actions d-flex justify-content-around total_amount">
+<div class='text general_text'>Total Amount: </div>
+<div class='headlines text'>$<%=@total_amount%>0 </div>
+</div>
+
+<div id="groups" class='general_text text'>
+  <% @groups.each do |group| %>
+  <%= link_to user_group_path(id: group.id) do%>
+  <div>
+  <% if can? :read, group %>
+    <%= render group %>
+
+  <% end %>
+  <% end %>
+  </div>
+  <% end %>
+</div>
+
+<div class="green d-flex flex-wrap justify-content-around headlines">
+<%= link_to "New category", new_user_group_path, class:"btn inter" %>
+</div>
```

Comment:
- [ ] [OPTIONAL] Your mobile design is flawless but I have noticed that the bigger the screen, the bigger the green button at the bottom of the page. As a result, the button is abnormally large on desktop devices. Please consider making it look better in order to enhance the UX 👍 
![image](https://user-images.githubusercontent.com/46839548/234640043-c57f64f3-d676-4101-a818-8ce4d61d007d.png)

**NOTE: This is an additional requirement.**